### DISCUSSION TITLE
Building for windows

### DISCUSSION THREAD

**Discussion by pvagner** [RHVoice/RHVoice#741] [Q&A]
Hello,
Perhaps this has already been discussed but I am unable to find it.
How are you usually building rhvoice for windows?
Which windows specific dependencies are required?
Has anyone tryed crosscompiling windows sapi5 version of rhvoice for windows with mingw on linux? Would that be doable?
If this is not possible, is there a way to build at least NVDA voice addons?
If nothing of this is possible is there a CI that would be capable of creating windows builds?

**Comment by zstanecic** [comment]
Hi,

As i know, the only way to build RHVoice on windows it to have physical windows machine. 

 

 

From: pvagner ***@***.***> 
Sent: Saturday, May 20, 2023 10:32 PM
To: RHVoice/RHVoice ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [RHVoice/RHVoice] Building for windows (Discussion #741)

 

Hello,
Perhaps this has already been discussed but I am unable to find it.
How are you usually building rhvoice for windows?
Which windows specific dependencies are required?
Has anyone tryed crosscompiling windows sapi5 version of rhvoice for windows with mingw on linux? Would that be doable?
If this is not possible, is there a way to build at least NVDA voice addons?
If nothing of this is possible is there a CI that would be capable of creating windows builds?
